adcom1/audio_placement.go

```diff
@@ -11,7 +11,7 @@ type AudioPlacement struct {
 	// Definition:
 	//   Indicates the start delay in seconds for pre-roll, mid-roll, or post-roll placements.
 	//   For additional generic values, refer to List: Start Delay Modes.
-	Delay int64 `json:"delay,omitempty"`
+	Delay StartDelayMode `json:"delay,omitempty"`
```

Comment:
Hmmm, I just noticed (grep-ed in code) that StartDelayMode was not used anywhere.

What I don't like is using enum-types for properties that hold seconds.

We either should:

- keep it as just int64 with special values (0, -1, -2) documented in property comment with maybe int64 (no special type) constants like "StartDelayPreroll" etc -> self-documenting code for lib users
- rename it type StartDelayMode -> StartDelay so its use is more natural - `StartDelay(seconds)` vs `StartDelayMode(seconds)` (it's not Mode in this case)

I'll revise later, good for now. Will change it myself, if decide to change it.

Reply:
I can understand how an int64 might be preferred. I don't have a strong opinion. It's a strange value.

Reply:
There was an `openrtb2.StartDelay` enum, maybe I'll bring it back (this time - in adcom1).